packages/notebook-extension/src/index.ts

```diff
@@ -2242,10 +2246,22 @@ function addCommands(
     isEnabled
   });
   commands.addCommand(CommandIDs.duplicateBelow, {
-    label: trans.__('Duplicate Cells Below'),
-    caption: trans.__(
-      'Copy the selected cells and paste them below the selection'
-    ),
+    label: args => {
+      const current = getCurrent(tracker, shell, args);
+      return trans._n(
+        'Duplicate Cell Below',
+        'Duplicate Cells Below',
+        current?.content.selectedCells.length ?? 0
+      );
+    },
+    caption: args => {
+      const current = getCurrent(tracker, shell, args);
+      return trans._n(
+        'Duplicate cell below',
+        `Duplicate cells below`,
```

Comment:
```suggestion
        'Create a duplicate of this cell below',
        `Create duplicates of these cells below`,
```

Reply:
Captions tend to be more verbose than labels are. Because "duplicate" can be both a noun or a verb in English (with different pronunciation but the same spelling), I think it would improve usability and localization success to use longer captions.

Reply:
@HaudinFlorence Sorry about the confusion! The only guidance I found about `label` vs. `caption` in the docs was [this section](https://jupyterlab.readthedocs.io/en/stable/extension/ui_helpers.html#using-commands) which describes `label` as "a short description" and `caption` as "a longer description". Perhaps the [writing style guide](https://jupyterlab.readthedocs.io/en/stable/developer/contributing.html#writing-style) could be modified to make this more explicit. I'll file a new issue about this.